tests/api/admin/test_password_admin_authentication_provider.py

```diff
@@ -34,6 +34,14 @@ def test_sign_in(self):
         assert PasswordAdminAuthenticationProvider.NAME == admin_details.get("type")
         assert "foo" == redirect
 
+        # Case insensitive test
+        admin_details, redirect = password_auth.sign_in(
+            self._db, dict(email="ADMin2@nyPL.Org", password="pass2", redirect="foo")
+        )
+        assert "admin2@nypl.org" == admin_details.get("email")
+        assert PasswordAdminAuthenticationProvider.NAME == admin_details.get("type")
+        assert "foo" == redirect
```

Comment:
It would probably be useful to add an analogous test in [TestSignInController.test_password_sign_in](https://github.com/ThePalaceProject/circulation/blob/7c9ae5b9f66b3f3ad26950412ae6da05aade7d16/tests/api/admin/controller/test_controller.py#L702-L715).

Reply:
@RishiDiwanTT 👆🏽

Reply:
Something more than this commit is required?
https://github.com/ThePalaceProject/circulation/pull/295/commits/50307f92bf1cd30a1e39e7698802ea2c8d58af2b

Reply:
I just went and looked and both the `/admin/sign_in` and `/admin/sign_in_with_password` routes end up in `PasswordAdminAuthenticationProvider.sig_in`, so yeah, this is fine.